graylog2-server/src/main/java/org/graylog/plugins/sidecar/migrations/V20180212165000_AddDefaultCollectors.java

```diff
@@ -96,9 +96,10 @@ private void ensureFilebeatCollectorsAndConfig() {
         StringBuilder filebeatConfigBuilder = new StringBuilder(f("""
                         %s
                         output.logstash:
-                           hosts: ["${user.graylog_host}:5044"]
-                        data: ${sidecar.spoolDir!\"/var/lib/graylog-sidecar/collectors/filebeat\"}/data
-                        logs: ${sidecar.spoolDir!\"/var/lib/graylog-sidecar/collectors/filebeat\"}/log
+                          hosts: ["${user.graylog_host}:5044"]
+                        path:
+                          data: ${sidecar.spoolDir!\"/var/lib/graylog-sidecar/collectors/filebeat\"}/data
+                          logs: ${sidecar.spoolDir!\"/var/lib/graylog-sidecar/collectors/filebeat\"}/log
```

Comment:
```suggestion
                          logs: ${sidecar.spoolDir!\"/var/lib/graylog-sidecar/collectors/filebeat\"}/log
                          
```
Since we are touching that anyway, can we add a newline here? 
The grouping is nicer this way.